components/log-viewer-webui/client/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "log-viewer-webui-client",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/index.jsx",
+  "scripts": {
+    "lint:check": "npx eslint --no-eslintrc --config package.json src webpack.config.js",
+    "lint:fix": "npx eslint --fix --no-eslintrc --config package.json src webpack.config.js",
+    "build": "webpack --define-process-env-node-env production",
+    "watch": "webpack serve"
+  },
+  "author": "YScope Inc. <dev@yscope.com>",
+  "license": "Apache-2.0",
+  "type": "module",
+  "devDependencies": {
+    "@babel/core": "^7.24.7",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
+    "@babel/preset-react": "^7.24.7",
+    "@webpack-cli/generators": "^3.0.7",
+    "autoprefixer": "^10.4.19",
+    "babel-loader": "^9.1.3",
+    "css-loader": "^7.1.2",
+    "eslint-config-yscope": "latest",
+    "html-webpack-plugin": "^5.6.0",
+    "mini-css-extract-plugin": "^2.9.0",
+    "prettier": "^3.3.2",
```

Comment:
Do we need to add a config file for this?

Reply:
Removed